driver/pelican/pelican.go

```diff
@@ -169,6 +193,43 @@ func (pel *Pelican) GetStatus() (*PelicanStatus, error) {
 		}
 	}
 
+	// Thermostat History Object Request to retrieve time stamp
```

Comment:
I think the principle we want to work with is if the thermostat is offline, then we don't want to give the impression that it is online. This should manifest itself in 2 ways:
- stop publishing data once it is "old" (e.g. more than 1 minute old)
- stop publishing liveness messages for this interface; this will discourage attempted actuation of the device

Reply:
Ahh okay, we can forget my long blurb then. Lol.

What do you think is the best way to stop publishing liveness messages? I assume you're referring to the lastalive metadata stuff? My understanding is that would require unsubscribing from the slots...? If so, this is possible with some recent changes I made in `bw2bind` that were merged in.